iron-overlay-behavior.html

```diff
@@ -492,6 +492,10 @@
      */
     _onCaptureClick: function(event) {
       if (!this.noCancelOnOutsideClick) {
+        if (this.withBackdrop) {
+          // Avoid click to go beyond the backdrop element.
+          event.preventDefault();
```

Comment:
question: do you think this is a duty of the manager? it is the one listening for `touchstart` after all..WDYT @notwaldorf ?